Common/Sources/Extensions/HyperwalletTransferMethod.swift

```diff
@@ -42,6 +42,12 @@ extension HyperwalletTransferMethod: GenericCellConfiguration {
                           "to".localized(),
                           getField(TransferMethodField.email.rawValue) ?? "")
 
+        case "VENMO_ACCOUNT":
```

Comment:
Can you update this to `TransferMethodType.venmoAccount.rawValue`. Also update other hardcode values for all cases with their respective enum rawtype as well.

Reply:
Done

Reply:
Code updated and pushed